### PR TITLE
Update Optim-Methods.md

### PR DESCRIPTION
**Pull request by helenlly**
## What changes were proposed in this pull request?
modify docs/docs/APIGuide/Optimizers/Optim-Methods.md/
1） update The styling of RMSProp 
2）update adagrad Parameters sections 

## How was this patch tested?
test local

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/2190

### CODE REVIEW

File: docs/docs/APIGuide/Optimizers/Optim-Methods.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ optimizer = Optimizer(
 ## RMSprop ##
 
 An implementation of RMSprop (Reference: http://arxiv.org/pdf/1308.0850v5.pdf, Sec 4.2)
+
 * learningRate : learning rate
 * learningRateDecaye : learning rate decay
```
Comment (hkvision) [inline docs/docs/APIGuide/Optimizers/Optim-Methods.md]: decaye -> decay

File: docs/docs/APIGuide/Optimizers/Optim-Methods.md [hunk 2]
```diff
@@ -171,6 +171,7 @@ optimizer = Optimizer(
...
 * decayRatee : decayRate, also called rho
```
Comment (hkvision) [inline docs/docs/APIGuide/Optimizers/Optim-Methods.md]: ratee -> rate

File: docs/docs/APIGuide/Optimizers/Optim-Methods.md [hunk 3]
```diff
@@ -276,6 +277,7 @@ large-scale stochastic problems, where opfunc is a noisy approximation of f(x).
 case, the learning rate allows a reduction of confidence in the step size.
 
 **Parameters:**
+
 * **maxIter** - Maximum number of iterations allowed. Default: 20
```
Comment (hkvision) [inline docs/docs/APIGuide/Optimizers/Optim-Methods.md]: It will be better to remove ** **?

**Comment by helenlly** [comment]
updated: decaye -> decay
               ratee->rate
remove  ** ** for some parameter words

**Pull request merged**